job/scriptsGen/normalizeSteps.js

```diff
@@ -48,8 +48,13 @@ function _normalizeNewFormatSteps(steps, defaultRuntime, onSuccess,
   onFailure, always, buildJobId, buildScriptsDir, buildStatusDir, group) {
   var clonedSteps = _.clone(steps);
   var defaultJobRuntime = _.clone(defaultRuntime) || {};
+
+  // TODO: Move setting defaults to rSync
```

Comment:
Let's remove the next TODO it was meant to be the same thing. Now we know where it needs to happen.